adafruit_midi/__init__.py

```diff
@@ -98,18 +102,20 @@ def in_channel(self, channel):
             raise RuntimeError("Invalid input channel")
 
     @property
-    def out_channel(self):
+    def out_channel(self) -> int:
         """The outgoing MIDI channel. Must be 0-15. Correlates to MIDI channels 1-16, e.g.
-        ``out_channel = 3`` will send to MIDI channel 4. Default is 0 (MIDI channel 1)."""
+        ``out_channel = 3`` will send to MIDI channel 4. Default is 0 (MIDI channel 1).
+        """
         return self._out_channel
 
     @out_channel.setter
-    def out_channel(self, channel):
+    def out_channel(self, channel: Optional[int]) -> None:
+        assert channel is not None
```

Comment:
I think I'm in favor of a more specific error message in the case of None. The assert here raises a fairly generic looking AssertionError but doesn't really mention what specifically was invalid. Somewhat related, it may be nice to include another sentence in the error string that says what the valid values are. Something like this maybe to cover both?
```
if channel is None or not 0 <= channel <= 15:
    raise RuntimeError("Invalid output channel. Must be 0-15")
```

Reply:
I was unable to determine why it wasn't feasible to make the argument of type `int`, not `Optional[int]`.

Reply:
latest commit changes the type to `int` without Optional.